src/views/AddPeopleAndRoles.vue

```diff
@@ -37,9 +37,19 @@ export default class AddPeopleAndRoles extends Mixins(EntityFilterMixin) {
 
   // Global actions
   @Action setAddPeopleRolesStepValidity!: ActionBindingIF
+  @Action setIgnoreChanges!: ActionBindingIF
 
   // Entity Enum
   readonly EntityTypes = EntityTypes
+
+  /** Called when component is created. */
+  private created (): void {
+    // ignore data changes until page has loaded
+    this.setIgnoreChanges(true)
+    Vue.nextTick(() => {
+      this.setIgnoreChanges(false)
+    })
+  }
```

Comment:
I found that some pages load and update the data in the period before they (and their children) are created and the next tick. (Note that 'mounted' is also called in there before the next tick.)